### ISSUE TITLE
simple stats

### ISSUE THREAD

**Issue by LinguList** [lexibank/lexitools#2]
I do not know to which degree simple statistics are already derived in lexibank, but things like "mutual coverage", useful for various purposes, could be calculated here without the lingpy dependency. If commands at some point turn out to be used frequently, one can then discuss moving them to pylexibank. 

Crucial for debugging are so far:

- mutual coverage
- synonymy (we have this in lexibank, but on a language basis in one table, it may be interesting)

I'll add more if I can think of more cases.

**Comment by xrotwang** [comment]
Well, currently, `lexitools` requires `pylexibank` - mostly because importing any dataset's python module will require `pylexibank` as well. So if we really wanted to drop this dependency, the commands would only have access to the CLDF dataset.

**Comment by LinguList** [comment]
No, no worries, this is fine. And in some sense one may even say:
lexitools can have more dependencies than the rest, right? But we should
already see to replace things that are in lingpy just because I did not
know where to put them with code more targeted for cldf wordlists, and
the like...
